Tensorflow installation problem

Installed Python 3.7.3 
NodeJs 10.16.0 LTS

When is try installing 
npm install --save technicalindicators

I've got this errors and installation not complete.
D:\NodeJSProjects\Lesson ----- is my local project folder

_______________________________________________________________


D:\NodeJSProjects\Lesson>npm install @tensorflow/tfjs-node

> @tensorflow/tfjs-node@1.1.2 install D:\NodeJSProjects\Lesson\node_modules\@tensorflow\tfjs-node
> node scripts/install.js

* Downloading libtensorflow
[==============================] 2730226/bps 100% 0.0s
* Building TensorFlow Node.js bindings
D:\NodeJSProjects\Lesson\node_modules\@tensorflow\tfjs-node\scripts\install.js:165
      throw new Error('node-gyp rebuild failed with: ' + err);
      ^

Error: node-gyp rebuild failed with: Error: Command failed: node-gyp rebuild
gyp ERR! configure error
gyp ERR! stack Error: Can't find Python executable "C:\Users\wissen\AppData\Local\Programs\Python\Python37\python.EXE", you can set the PYTHON env variable.
gyp ERR! stack     at PythonFinder.failNoPython (C:\Users\wissen\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\lib\configure.js:483:19)
gyp ERR! stack     at PythonFinder.<anonymous> (C:\Users\wissen\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\lib\configure.js:508:16)
gyp ERR! stack     at C:\Users\wissen\AppData\Roaming\npm\node_modules\npm\node_modules\graceful-fs\polyfills.js:284:29
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:153:21)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\wissen\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "
rebuild"
gyp ERR! cwd D:\NodeJSProjects\Lesson\node_modules\@tensorflow\tfjs-node
gyp ERR! node -v v10.16.0
gyp ERR! node-gyp -v v3.6.2
gyp ERR! not ok

    at cp.exec (D:\NodeJSProjects\Lesson\node_modules\@tensorflow\tfjs-node\scripts\install.js:165:13)
    at ChildProcess.exithandler (child_process.js:301:5)
    at ChildProcess.emit (events.js:198:13)
    at maybeClose (internal/child_process.js:982:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)

npm WARN enoent ENOENT: no such file or directory, open 'D:\NodeJSProjects\Lesson\package.json'
npm WARN rollup-plugin-visualizer@1.1.1 requires a peer of rollup@>=0.60.0 but none was installed.
npm WARN Lesson No description
npm WARN Lesson No repository field.
npm WARN Lesson No README data
npm WARN Lesson No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @tensorflow/tfjs-node@1.1.2 install: `node scripts/install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @tensorflow/tfjs-node@1.1.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\wissen\AppData\Roaming\npm-cache\_logs\2019-06-11T15_50_40_061Z-debug.log

--- a/package.json
+++ b/package.json
@@ -147,8 +147,8 @@
     "typescript": "^2.9.2"
   },
   "dependencies": {
-    "@tensorflow/tfjs": "^0.13.0",
-    "@tensorflow/tfjs-node": "^0.1.17",
+    "@tensorflow/tfjs": "^0.13.5",
+    "@tensorflow/tfjs-node": "^0.3.2",
     "@types/node": "^6.0.96"
   }
 }

